pkg/mapper/mapper.go

```diff
@@ -75,6 +81,20 @@ var defaultQuantiles = []metricObjective{
 	{Quantile: 0.99, Error: 0.001},
 }
 
+func min(x, y int) int {
```

Comment:
I think these are moved into the FSM now too?

Reply:
oh yeah, sorry 😅